scipy/stats/morestats.py

```diff
@@ -3478,3 +3478,76 @@ def circstd(samples, high=2*pi, low=0, axis=None, nan_policy='propagate'):
         R = np.minimum(1, hypot(sin_mean, cos_mean))
 
     return ((high - low)/2.0/pi) * sqrt(-2*log(R))
+
+
+def quartile_coeff_dispersion(a, q=(0.25, 0.75), axis=0,
+                              interpolation='lower'):
+    """
+    Compute the quartile coefficient of dispersion for a given sample array.
+
+    Given two quartiles Q_low, Q_high from array samples such that
+    Q_low < Q_high, the quartile coefficient of dispersion is calculated by
+    (Q_high - Q_low)/(Q_high + Q_low).
+    by default it is computed using 3rd and 1st quartiles.
+
+    Parameters
+    ----------
+    a : array_like
+        Input array.
+    q : array_like of float, optional
+        Sequence of 2 quantiles, such that q[0] < q[1].
+        The default is to compute with Q_high = 0.75, Q_low = 0.25.
+    axis : {int, tuple of int, None}, optional
+        Axis along which the coefficient of dispersion is computed.
+        Default is 0. If None, compute over the whole array a.
```

Comment:
```suggestion
        Default is 0. If None, compute over the whole array `a`.
```
Referenced parameters [get single backticks](https://numpydoc.readthedocs.io/en/latest/format.html). (Code gets double backticks.)

I think "Q_low" and "Q_high`" are fine as they are because they are neither really code nor parameters.